moove/commons/src/main/kotlin/io/charlescd/moove/commons/integration/git/service/GitHubServiceLegacy.kt

```diff
@@ -107,14 +117,23 @@ class GitHubServiceLegacy(private val gitHubClientFactoryLegacy: GitHubClientFac
                 log.info("release: $releaseName created successfully")
             }
         } catch (e: Exception) {
-            log.error("failed to create release: $releaseName with error: ${e.message}")
-            handleResponseError(
-                error = e,
-                repository = repository,
-                baseBranch = sourceBranch,
-                releaseName = releaseName
-            )
-            Optional.empty()
+            return try {
+                Optional.of(
+                    repositoryService.createRelease(repository, alternativeBaseBranch, releaseName, description).get("name")
+                        .asString
+                ).apply {
+                    log.info("release: $releaseName created successfully")
+                }
+            } catch (e: Exception) {
+                log.error("failed to create release: $releaseName with error: ${e.message}")
+                handleResponseError(
+                    error = e,
+                    repository = repository,
+                    baseBranch = sourceBranch,
```

Comment:
Should be equal the above method 
```suggestion
                    baseBranch = "$sourceBranch/$alternativeBaseBranch",
```